aquifer/aquifer
# Update tools, get ready for es6!

This PR updates our toolsets in a couple ways:
1. Updates gulpfile to use es6. We are going to start using es6 in aquifer, a clean up and es6-ification PR is coming soon in preparation for a couple things that could really implement some of es6's features.
2. Replace jscs with eslint completely. Everything jscs can do, eslint can do, or at least there's a lot of overlap.
3. Execute `gulp lint` on `npm run lint`. Eventually, I'd like to steer away from gulp. that's a whole different discussion, but for now, it's good to have those tasks defined as npm scripts.
## Steps to test
- Checkout this branch.
- Run `gulp lint`
- Run `npm run lint`
- Make sure everything works. YAY!
